### PR TITLE
Update Offline Repo steps

### PR DESCRIPTION
**Pull request by jgrund**
The Offline repo steps weren't clear enough
in that user's need to switch to the branch /
tag of the release they want to build for.

  - Fix instructions to be clearer.
  - Rename file to drop cd_ prefix.
  - Fix some linting issues.

Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: docs/Create_Offline_Repos_10_0.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# How to create offline repos for IML {{site.version}}
+
+[**Online Help Table of Contents**](IML_Help_TOC.md)
+
+The following is a procedure for creating local repos using a CentOS 7 VM.
+
+1.  Check out a local copy of IML from github.
```
Comment (johnsonw) [inline docs/Create_Offline_Repos_10_0.md]: Might be useful to provide the git url: https://github.com/whamcloud/integrated-manager-for-lustre.git

File: docs/Create_Offline_Repos_10_0.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+1.  Navigate to the IML project dir.
+
+1.  Switch to the tag or branch you want to create offline repos for. For example, if building offline repos for IML 4.0.6:
```
Comment (johnsonw) [inline docs/Create_Offline_Repos_10_0.md]: I know this is an example but we could probably replace 4.0.6 with {{site.version}}. Thoughts?

File: docs/Create_Offline_Repos_10_0.md (hunk 1 above)
Comment (jgrund) [reply docs/Create_Offline_Repos_10_0.md]: Sure.

File: docs/Create_Offline_Repos_10_0.md (hunk 2 above)
Comment (jgrund) [reply docs/Create_Offline_Repos_10_0.md]: I purposely did it this way to illustrate we can use multiple older versions.

**Pull request merged**